packages/python-pulp-deb/python-pulp-deb.spec

```diff
@@ -1,36 +1,38 @@
-# Created by pyp2rpm-3.3.2
+# Created by pyp2rpm-3.3.3
 %global pypi_name pulp-deb
 
 Name:           python-%{pypi_name}
-Version:        2.4.0
-Release:        0.1.b1%{?dist}
+Version:        2.6.1
+Release:        1%{?dist}
 Summary:        pulp-deb plugin for the Pulp Project
 
 License:        GPLv2+
-URL:            https://pulpproject.org/#deb
-Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}b1.tar.gz
+URL:            https://pulpproject.org
+Source0:        https://files.pythonhosted.org/packages/source/p/%{pypi_name}/%{pypi_name}-%{version}.tar.gz
```

Comment:
@evgeni Any idea why pyp2rpm doesn't use `%{pypi_source}`? It'd be much nicer.

Reply:
I think I disabled that in our template as it didn't work for the modules we rename. Or something like that. we can revisit that